spec/requests/api/editor/editor_can_see_all_articles_spec.rb

```diff
@@ -3,11 +3,12 @@
   let!(:journalist) { create(:user, role: 'journalist') }
   let!(:journalist2) { create(:user, role: 'journalist') }
   let(:auth_headers) { editor.create_new_auth_token }
-
+  
   describe 'successfully' do
     let!(:article1) { create(:article, title: 'First Article', user_id: journalist.id) }
     let!(:article2) { create(:article, title: 'Second Article', user_id: journalist.id) }
     let!(:article3) { create(:article, title: 'Third Article', user_id: journalist2.id) }
+    let!(:comment) { create(:comment, article_id: article3.id) }
```

Comment:
might wanna rename them to "something_num.id" instead as You Know Who said earlier today? ;)

Reply:
Iz true. Let's sort that in our big refactor session :3